### ISSUE TITLE
Monero Community Workgroup Meeting: Saturday 17th August 15:00UTC

### ISSUE THREAD

**Issue by plowsof** [monero-project/meta#1055]
Location: [Libera.chat, #monero-community](https://libera.chat/) | [Matrix](https://matrix.to/#/#monero-community:monero.social?via=matrix.org&via=monero.social)

[Instructions for joining the monero.social Matrix server.](https://www.getmonero.org/resources/user-guides/join-monero-matrix.html)

Time
15:00 UTC [Check your timezone](https://www.timeanddate.com/worldclock/converter.html)

Moderator: plowsof

Please reach out in advance of the meeting if you would like to propose an agenda item.

Proposed Meeting Items:

1. Introduction
2. Greetings
3. Community highlights    
    - featherwallet 2.6.8 released https://featherwallet.org/changelog/
    - [[WIP] FCMP++ integration](https://github.com/monero-project/monero/pull/9436) - j-berman
    - TownForge testnet to restart on 1st September with [Proof Of Settlement](https://townforge.net/proof-of-settlement)
News: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/)- [Monero Moon](https://www.themoneromoon.com/)    
5. [CCS updates](https://ccs.getmonero.org/)   
  a. [From Prototype to Marketplace: Maturing the XMR-BTC Atomic Swaps Ecosystem
](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/477) Merged to funding but has an error somewhere, however, lederstrumpf of farcaster left an [important comment](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/477#note_25710) which we are ~~waiting for a response to before putting to funding.~~ 
  a.1 [Rate change after merge*](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/485)
  b. [New Monero Website](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/450)    
  c. [ofrnxmr support, docs, site, meta++ (totw pt3)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/482) After the merging / quick funding of ofrnxmrs BSX proposal what to do with this proposal? some thoughts:
      - Merge it as the tasks listed are unrelated to the bsx proposal
      - Ask the updooters if the bsx merge changes anything
      - Wait to see how ofrnxmr handles the bsx in-progress ccs
    
   d. [hinto-janai full-time work on Cuprate (3 months)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/484)    
   e. [0xfffc 3 months full-time dev.](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/487)    

6. Workgroup reports    
  a. Dev workgroup
  b. Localization workgroup
  c. Outreach workgroup
  d. Events workgroup - [MoneroKon 2024](https://www.MoneroKon.org)
  e. Website workgroup
  f. Policy workgroup
  g. Research workgroup
  h. [Seraphis Migration workgroup](https://github.com/seraphis-migration)
7. Open ideas time    
8. Confirm next meeting date/time    

[Previous meeting including logs](https://github.com/monero-project/meta/issues/1046)    

Meeting logs will be posted here afterwards.

**Comment by plowsof** [comment]
Logs 
> __< p​lowsof:matrix.org >__ binarybaron+einlitre are currently having internal discussions with lederstrumpf (of farcaster) who has proposed a sane path forward for integrating farcaster swaps into the unstoppableswaps gui. so in the end it would support both COMIT and Farcaster.  the unstoppable swaps proposal could possibly be edited to include these extra tasks, presumable with an adjustment in rates/amounts to account)     

> __< p​lowsof:matrix.org >__ pending binarybarons feedback on this^ i would like to suggest a separate scoped out ccs proposal with concrete deliverables for adding farcaster into the unstoppableswap gui     

> __< o​frnxmr:monero.social >__ butbutbut "month 1, month 2, month 3" are so much better..     

> __< o​frnxmr:monero.social >__ about to be 70hr/week/person 🙃     

> __< p​lowsof:matrix.org >__ lederstrumpf has provided them a route to doing this where most of the development work has already been done, i can't speak to that nor say how binarybaron feels about it     

> __< o​frnxmr:monero.social >__ I think farcaster is out of scope, as is the last minute rate change. Proposal was merged as-is. We'd be just as wrong to "unmerge" it, as they were to modify it post-merge     

> __< o​frnxmr:monero.social >__ the "contract" was officially accepted as-proposed when ouji hit "merge". Fortunately/unfortunately, lederstrumpf was late 2 the party     

> __< o​frnxmr:monero.social >__ i know of a few other, unrealistic, unachievable, or straight up scam ccs that would be or would have been more logical to "take back", but that is not an option.     

> __< o​frnxmr:monero.social >__ We even paid out some of these, after they proved they wouldn't deliver     

> __< o​frnxmr:monero.social >__ If unstoppable is adding farcaster, they should open a new, parallel ccs     

> __< o​frnxmr:monero.social >__ bUt ShoUlD wAiT tO seE hOw ThE fIrSt OnE gOeS bEforE wE cAn mErGe tHe 2nD     

> __< plowsof >__ https://libera.monerologs.net/monero-community/20240817#c412195 pending binarybarons feedback on discussions      

> __< o​frnxmr:monero.social >__ If not for the weak ass "lets payout incomplete / scam ccs", merging multiple wouldn't be an issue. No completion = thanks for donating to jet fund. There is 0 risk im raising $ unless you plan on being bullied / peer pressured into paying it out     

> __< o​frnxmr:monero.social >__ > a separate scoped out ccs proposal with concrete deliverables for adding farcaster into the unstoppableswap gui     

> __< o​frnxmr:monero.social >__ this has no bearing on already-merged 477. Its a separate ccs     

> __< o​frnxmr:monero.social >__ Unmerging is dirty dictator stuff. Changing rates post-merge is laughable as well. If farcaster integration ir rate changes are a thing, they should have been brought up before the meetings and before the merge. Changes now must come in the form of a new proposal     

> __< nioCat >__ is it separate or would it significantly better to be done in parallel?  I have no way of knowing but this seems to be a question to be answered       

> __< plowsof >__ yes, separate ccs, the rates change is the issue      

> __< plowsof >__ i too await binarybarons thoughts      

> __< plowsof >__ what does this mean "There is 0 risk im raising $ unless you plan on being bullied / peer pressured into paying it out"     

> __< o​frnxmr:monero.social >__ separate ccs, parallel work     

> __< o​frnxmr:monero.social >__ Raising $ != paying money     

> __< o​frnxmr:monero.social >__ We tend to pay out ppl who dont finish their work (like mj)     

> __< o​frnxmr:monero.social >__ If we stopped doing that, there is 0 risk in merging. No work = no pay     

> __< nioCat >__ so separate but parallel,  seems like semantics      

> __< nioCat >__ but as plowsof says we need to wait for more info     

> __< o​frnxmr:monero.social >__ the first ccs was merged. Unmerging is as unprofessional as changing ratea     

> __< nioCat >__ what is that status of the rate changes     

> __< o​frnxmr:monero.social >__ The first ccs was accepted with no opposition until _after_ it was merged     

> __< o​frnxmr:monero.social >__ Status of rate change, if youre asking my view = lmao. No.     

> __< plowsof >__ would the community like farcaster in unstoppableswap gui (basically) combined with, are the gui devs comfortable with the approach. lederstrumpf is also not opposing the ccs, merely suggesting (and correcting their, at times, incorrect knowledge) of farcaster      

> __< nioCat >__ it was changed but not changed back?     

> __< plowsof >__ geonics comment today https://libera.monerologs.net/monero-community/20240817#c412137     

> __< plowsof >__ MR to adjust the rates (+57 extra) https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/485     

> __< o​frnxmr:monero.social >__ https://libera.monerologs.net/monero-community/20240816#c412127 plowsof's comments make the mose sense     

> __< nioCat >__ "actually an hour and 20 minutes after luigi merged"  <> how is this possible?     

> __< plowsof >__ proposal merged with errors in the front matter so it did not display on the funding page     

> __< o​frnxmr:monero.social >__ They opened 485 AFTER luigi merged     

> __< plowsof >__ the errors where caused by my 'create a proposal script' BTW      

> __< plowsof >__ shame on me, corrected now + added a check in the pre parser      

> __< nioCat >__ the original guideline of 3 month max was due in part to volatility.  Back when there was significantly more volatility.      

> __< nioCat >__ we are stable coin now  lol     

> __< o​frnxmr:monero.social >__ 1. Leave 477 as is. No rate change. No scope change. Farcaster stuff is a new ccs / new milestone, and might not be something BB is even interested in     

> __< o​frnxmr:monero.social >__ 2. Set bad precedent and allow and/or require people to change their proposals after merge     

> __< o​frnxmr:monero.social >__ Sorry, A and B instead 1 and 2     

> __< plowsof >__ do we even have a precedent anymore      

> __< o​frnxmr:monero.social >__ No, we have rules for some ppl that dont apply to others     

> __< o​frnxmr:monero.social >__ Failed ccs >> invited to open new ccs (molly > sdk).     

> __< o​frnxmr:monero.social >__ In progress ccs (diego) > invited to open new ccs for website design     

> __< o​frnxmr:monero.social >__ in progress ccs (ofrn) > told to pause monero work to see how bsx goes      

> __< o​frnxmr:monero.social >__ "no upfront" but fund movies wirh 50:50 votes and 50% upfront     

> __< o​frnxmr:monero.social >__ Allowing solopt to change their milestones and allowing the project lead to be faid in full before completion     

> __< p​lowsof:matrix.org >__ molly / valdrac still making progress, this merge completed one of his milestones https://github.com/monero-project/monero/pull/9344     

> __< o​frnxmr:monero.social >__ Project lead claimed he'd complete the project, yet changed milestones so that he could be paid to abandoned it     

> __< p​lowsof:matrix.org >__ i think diegos website design might fall into his current ccs (can't pull the comment up) -site meeting tomorrow though     

> __< o​frnxmr:monero.social >__ Diegos was just an example, he doesnt have a second ccs open     

> __< p​lowsof:matrix.org >__ "in progress ccs (ofrn) > told to pause monero work to see how bsx goes" on the meeting agenda in.. 15 mins ~ https://github.com/monero-project/meta/issues/1055#issue-2463763769     

> __< p​lowsof:matrix.org >__ r4v3r23: any anonero updates?     

> __< o​frnxmr:monero.social >__ ^^ perfect example of ridiculousness. Telling me to pause work, and to continue campaigning instead     

> __< o​frnxmr:monero.social >__ meanwhile mb and diego have copy pasta ccs and site still has no maintainer     

> __< o​frnxmr:monero.social >__ i'm basicallt expected to stop all contributions, whether that be stressnet, site or docs, and to go on twitter amd try to rally another 30 upvotes so i can wait 3 more months for luigi to get outta my way so we can get work done 🙃, while dealing with yasabi and his socks bot spamming ccs and community. Fun     

> __< p​lowsof:matrix.org >__ hello einliterflasche2     

> __< d​iego:cypherstack.com >__ Meeting in 5 yes?     

> __< p​lowsof:matrix.org >__ yes     

> __< nioCat >__ having more than 1 CCS at the same time would not be precedent setting.  AIUI this happens with proven CCS contributors.  ofrn's 2nd CCS doesn't seem to be a risk at all.  The first one I could not wrap my head around but it is now a done deal     

> __< nioCat >__ <d​iego:cypherstack.com> Meeting in 5 yes? <> we started early because 1 hr is too short for a meeting  ( ͡° ͜ʖ ͡°)     

> __< p​lowsof:matrix.org >__ early meeting time https://github.com/monero-project/meta/issues/1055     

> __< plowsof >__ hello all, greetings      

> __< msvb-lab >__ Hello.     

> __< plowsof >__ just reminder that a new monero version was tagged, and reproducible build hashes already being submitted, so a release will follow shortly      

> __< plowsof >__ featherwallet has already bumped to the latest monero versions in the new release https://featherwallet.org/changelog/      

> __< l​ederstrumpf:matrix.org >__ I struggle to see how either sets bad precedent: the exceptional scenario here is that the MR was merged but did not actually open up for funding on website due to a bug. So this doesn't set a precedent for normal CCS operation. If the funding had already been opened and funds contributed, I'd be more inclined to agree with you since then it would be a change of the terms against <clipped message>     

> __< l​ederstrumpf:matrix.org >__ what donors had signed up for.     

> __< plowsof >__ TownForge testnet to restart on 1st September with Proof Of Settlement https://townforge.net/proof-of-settlement      

> __< o​frnxmr:monero.social >__ The last part of your sentence is the point     

> __< o​frnxmr:monero.social >__ K lets follow agenda     

> __< l​ederstrumpf:matrix.org >__ but no one could donate to it yet?     

> __< d​iego:cypherstack.com >__ Yo yo. Meeting time.     

> __< d​iego:cypherstack.com >__ Plowsof lez go     

> __< p​lowsof:matrix.org >__ merged into limbo with no donation address     

> __< o​frnxmr:monero.social >__ agenda     

> __< p​lowsof:matrix.org >__ has anything else amazing happened in the ecosystem amazing happened in the last 7 days other than us all being here     

> __< d​iego:cypherstack.com >__ Yes Cypher Stack is about to release a summary of current knowledge of sorts on Monero. Today or Monday.     

> __< d​iego:cypherstack.com >__ Started design process for the website.     

> __< d​iego:cypherstack.com >__ Been pushing forward on Astro learning.     

> __< m​onerobull:matrix.org >__ haveno update is live     

> __< o​frnxmr:monero.social >__ Design.getmonero.org     

> __< m​onerobull:matrix.org >__ they have pow nodes now     

> __< d​iego:cypherstack.com >__ Oh yeah and we got penpot up for you guys to play with. Thanks.     

> __< p​lowsof:matrix.org >__ The recent MRL meeting shared a nice research paper, with some interesting acknowledgements, Rucknium chaired the meeting     

> __< p​lowsof:matrix.org >__ monerobull : using the equiX thing from tevador? its enabled by default for everyone now or something?     

> __< m​onerobull:matrix.org >__ yeah with tor pow     

> __< m​onerobull:matrix.org >__ its enabled but you only do pow while there is an attack     

> __< o​frnxmr:monero.social >__ For all nodes or just seed nodes?     

> __< p​lowsof:matrix.org >__ nice, so this should hopefully reduce the effects of any future DDOS attacks on any haveno network instances #haveno:monero.social #haveno-lounge:monero.social  #haveno-dev:haveno.network     

> __< m​onerobull:matrix.org >__ well you can just delete tor files should your personal onion actually get attacked     

> __< p​lowsof:matrix.org >__ and people are looking for the mysu source code (?)     

> __< r​ottenwheel:kernal.eu >__ Yes, v0.18.3.4 point release and we can't have someone else upgrade it, because no code. Pooookkst.     

> __< r​ottenwheel:kernal.eu >__ Summary of current knowledge of sorts on Monero? 🤔 What exactly is that?     

> __< d​iego:cypherstack.com >__ Nice chunky 24 page paper with tons of references.     

> __< r​ottenwheel:kernal.eu >__ On what exactly? What is it?     

> __< r​ottenwheel:kernal.eu >__ Summary of current knowledge of sorts on Monero is too vague.     

> __< r​ottenwheel:kernal.eu >__ A new moneropedia? Lol.     

> __< p​lowsof:matrix.org >__ probably something for researchers / devs. they can digest it in the -dev meeting that identifies as the -nwlb meeting sometimes     

> __< o​frnxmr:monero.social >__ Oh that     

> __< d​iego:cypherstack.com >__ In fact: https://github.com/cypherstack/pup-monero-analysis/releases/tag/final     

> __< o​frnxmr:monero.social >__ The meeting are moving back to nwlb, but nwlb will be readdressed as monero-somethinf     

> __< d​iego:cypherstack.com >__ Literally hot off the press.     

> __< p​lowsof:matrix.org >__ full-ish picture of this week i think? please to check the news sources for a more comprehensive picture News: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/)- [Monero Moon](https://www.themoneromoon.com/)     

> __< o​frnxmr:monero.social >__ names such as monero-next monero-dev-lab and monero-tech have been proposed. I don't like monero-tech, because of monerotech.info     

> __< o​frnxmr:monero.social >__ https://github.com/monero-project/meta/issues/1058     

> __< p​lowsof:matrix.org >__ more regular meetings for devs in the immediate short team does seem beneficial     

> __< p​lowsof:matrix.org >__ s/team/term     

> __< p​lowsof:matrix.org >__ thanks or the meta issue link     

> __< o​frnxmr:monero.social >__ https://github.com/monero-project/meta/issues/1054 and thr actual discussion here     

> __< p​lowsof:matrix.org >__ ok lets jump into the ccs merges list unless theres an immediate opinion being typed out     

> __< o​frnxmr:monero.social >__ The main thing we need now for nwlb, is a name, and for plowsof to add matrix.org address after we choose one     

> __< o​frnxmr:monero.social >__ monero-dev can wait for now, with input from selsta etc     

> __< d​iego:cypherstack.com >__ Merge list. Weeee!     

> __< p​lowsof:matrix.org >__ a. [From Prototype to Marketplace: Maturing the XMR-BTC Atomic Swaps Ecosystem     

> __< p​lowsof:matrix.org >__ ](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/477)     

> __< p​lowsof:matrix.org >__ so this is the one which was merged, with errors in the front matter preventing it from displaying on the funding page. a comment from ledertrumpf containing some corrections reg comments about farcaster was made, and seemed important enough to hold off on a response. now..     

> __< p​lowsof:matrix.org >__ a.1 [Rate change after merge*](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/485)     

> __< binarybaron >__ The rate change was discussed with Luigi privately (its common for luigi to ping the authors of a proposal to update the rates before merging which he didnt do with us). If it's not acceptable to update them know we'd be fine with sticking with the old exchange rate.     

> __< binarybaron >__ now*     

> __< d​iego:cypherstack.com >__ I do know people are often asked to update rates before merge.     

> __< o​frnxmr:monero.social >__ That seems settled.     

> __< o​frnxmr:monero.social >__ what are your thoughts on a new ccs for farcaster comments?     

> __< p​lowsof:matrix.org >__ 1. merge the above rate change no questions asked and put to funding with +57 (no mention of farcaster)     

> __< p​lowsof:matrix.org >__ 2. put the original proposal as is to the funding page     

> __< p​lowsof:matrix.org >__ 3. let lederstrumpf cook some more with his path to farcaster integration inside unstoppableswap and include it in the proposal / hear from binary baron about his thoughts      

> __< p​lowsof:matrix.org >__ 4. again hear BB's thoughts - but on a seperate farcaster integration proposal (if this is even possible with the proposed path to integration which supposedly removes alot of the development effort required)     

> __< p​lowsof:matrix.org >__ other options? edits to those ones?     

> __< o​frnxmr:monero.social >__ Something youd be interested in, or not interested?     

> __< o​frnxmr:monero.social >__ 2 and 4     

> __< l​ederstrumpf:matrix.org >__ wrt rate: my experience with other crypto ecosystems is also that if the exchange rate moves materially until funding actually opens, it is adjusted prior to actually funding. And to restate my prior comment to ofrnxmr: since no one could actually donate to it, I personally don't think it's an issue since no-one contributed on the terms of the current outdated rate.     

> __< o​frnxmr:monero.social >__ well, i was unlucky to be merged in the middle of the night     

> __< o​frnxmr:monero.social >__ And funded soon after     

> __< o​frnxmr:monero.social >__ I want to adjust my rates /s     

> __< r​ottenwheel:kernal.eu >__ Would look good on binarybaron if he replied to leder's comment and we potentially benefitted from that back and forth. If not for them to work together, for binarybaron's project to become yet stronger and more likely to occur.     

> __< p​lowsof:matrix.org >__ there is some context behind selecting number 1 : some discussion here geonics comment today https://libera.monerologs.net/monero-community/20240817#c412137 and  https://libera.monerologs.net/monero-community/20240816#c412121     

> __< o​frnxmr:monero.social >__ You have days between meeting and merge     

> __< r​ottenwheel:kernal.eu >__ lederstrumpf Can you share what happened with hashed, zkao and the others from farcaster team? They disappeared after completing that CCS proposal of yours.     

> __< r​ottenwheel:kernal.eu >__ Vaguely remember playing with zkao in -swaps because I was going to test farcaster when ready. :-( I thought we had something special going on!     

> __< o​frnxmr:monero.social >__ The consensus was publicly announced and accepted. As plowsof said, its a longer (6 month) ccs that should have absorbed volatility     

> __< binarybaron >__ We put a lot of thought into our proposal and we think it's the best path to work on COMIT first and later expand to other protocols such as farcaster. we support 1.     

> __< l​ederstrumpf:matrix.org >__ I see binarybaron has provided an update on the CCS in the meantime, but that was before I spelled out my proposal for a risk minimized route of going farcaster first to them in DM     

> __< p​lowsof:matrix.org >__ binarybaron is OK with the old rates, but, lederstrumpfs comment is correct, it was not put to funding, with a donation address     

> __< p​lowsof:matrix.org >__ lederstrumpfs comment: as you see its not something to type in a few mins, he gave a headsup he was writing it*     

> __< o​frnxmr:monero.social >__ It didint have a donation t addr, but it was accepted and commited to the repo. The page went live but had a formatting error     

> __< o​frnxmr:monero.social >__ If this happened eith my proposal, i could have added 8% increase to it     

> __< d​iego:cypherstack.com >__ I think there's much ado about nothing here.     

> __< d​iego:cypherstack.com >__ They asked Luigi if they could update rates, and Luigi said yes.     

> __< l​ederstrumpf:matrix.org >__ I don't know what specifically zkao and h4sh3d are working on these days. TheCharlatan's doing bitcoin core dev - you can follow his work online.     

> __< o​frnxmr:monero.social >__ thats not what luigi told me, but ok     

> __< d​iego:cypherstack.com >__ Luigi told you they didn't ask? Or he told you you couldn't adjust rates?     

> __< binarybaron >__ Switching to Farcaster would be a drastic and risky change. We are confident that we can execute our goals by sticking with COMIT and later thinking about integrating Farcaster. We have put a lot of thought into this.     

> __< l​ederstrumpf:matrix.org >__ and it's pretty clear what kayabanerve's working on these days :P     

> __< o​frnxmr:monero.social >__ The former     

> __< d​iego:cypherstack.com >__ It's clear to me that farcaster needs to be a separate CCS.     

> __< p​lowsof:matrix.org >__ is the proposal a one-off thing as einlitre stated on the proposal https://libera.monerologs.net/monero-community/20240816#c412128     

> __< d​iego:cypherstack.com >__ Gotcha. So let's table until we hear from him. I pinged him privately.     

> __< binarybaron >__ We definitely did ask Luigi. He told us it was okay to change them before the formatting issue got fixed.     

> __< p​lowsof:matrix.org >__ or shuld we seek another team to look at integrating?     

> __< o​frnxmr:monero.social >__ +1 diego     

> __< l​ederstrumpf:matrix.org >__ please reconsider with the proposal I sent you this afternoon: again, by going for that first, you save duplicating components existing in Farcaster but not yet in COMIT.     

> __< d​iego:cypherstack.com >__ Sorry binarybaron, give us just a bit more patience. If Luigi confirms I have no issue with 1.     

> __< d​iego:cypherstack.com >__ Farcaster needs to make it's own proposal here.     

> __< d​iego:cypherstack.com >__ My thoughts at least. Maybe next?     

> __< p​lowsof:matrix.org >__ seems so     

> __< p​lowsof:matrix.org >__ b. [New Monero Website](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/450)     

> __< o​frnxmr:monero.social >__ Design.getmonero.org     

> __< p​lowsof:matrix.org >__ are we deferring this to site meeting tomorrow or is this a merge     

> __< o​frnxmr:monero.social >__ #monero-site has a meeting tomorrow     

> __< d​iego:cypherstack.com >__ This can't be merged. Design isn't done. Next.     

> __< o​frnxmr:monero.social >__ https://github.com/monero-project/meta/issues/1049     

> __< p​lowsof:matrix.org >__ thanks ok     

> __< o​frnxmr:monero.social >__ Defer. Cant merge cuz we have no idea what the new scope is     

> __< d​iego:cypherstack.com >__ Janaka doesn't even want it merged until design is done so they can come up with a good quote for making it.     

> __< d​iego:cypherstack.com >__ It was merely reopened to show that it can be merged in the future and wouldn't get lost in the weeds.     

> __< p​lowsof:matrix.org >__ yes, makes sense     

> __< o​frnxmr:monero.social >__ monero-site has taken over the decisions on this one, i think they need to come to consensus first     

> __< p​lowsof:matrix.org >__ moving onward     

> __< p​lowsof:matrix.org >__ c. [ofrnxmr support, docs, site, meta++ (totw pt3)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/482)     

> __< x​mrscott:monero.social >__ If [events] have Core so spoked they're editing terms post merge for part 2, shouldn't do part 3 IMO     

> __< o​frnxmr:monero.social >__ Huh     

> __< d​iego:cypherstack.com >__ Which specifically?     

> __< o​frnxmr:monero.social >__ Anyway, ignoring this guy     

> __< o​frnxmr:monero.social >__ Spooked?     

> __< o​frnxmr:monero.social >__ 👻     

> __< x​mrscott:monero.social >__ Escrow with things like this: https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/457#note_25742     

> __< p​lowsof:matrix.org >__ my latest comment to clarify things here https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/457#note_25800     

> __< 0​xfffc:monero.social >__ Merge.     

> __< s​yntheticbird:monero.social >__ I've been absent for almost 3 weeks and everytime I come back ofrnxmr CCS(s) is(are) more a brainfuck than ever.     

> __< d​iego:cypherstack.com >__ This seems to be a misunderstanding of what we presented in the proposal.     

> __< d​iego:cypherstack.com >__ If I may elaborate as I see it.     

> __< s​yntheticbird:monero.social >__ not saying it shouldn't be merged but having to go over 80+ comments to understand is awful     

> __< 0​xfffc:monero.social >__ Just look at the amount of time he is spending on helping others in monero / -community / -stressnet.     

> __< d​iego:cypherstack.com >__ ofrnxmr requested as part of the proposal that plowsof escrow ALL of the funds of the proposal pretty much at once. This is because he's been very critical of CCS wallet being drained and doesn't trust luigi to not lose the money.     

> __< d​iego:cypherstack.com >__ this request was denied     

> __< d​iego:cypherstack.com >__ That part of the m0 milestone go to plowsof to disperse isn't the same ask.     

> __< o​frnxmr:monero.social >__ Denied >> plowsof didnt want to     

> __< d​iego:cypherstack.com >__ The m0 is being paid out anyways, so it doesn't matter much who 'escrows' that funds.     

> __< d​iego:cypherstack.com >__ No. luigi denied also very plainly.     

> __< x​mrscott:monero.social >__ I do continue to maintain that if Core has knowledge of ofrnxmr engaging in malicious actions against the community and is paying them off, it would be quite bad for the project     

> __< p​lowsof:matrix.org >__ that i can't hold 2k monero, not surprising     

> __< o​frnxmr:monero.social >__ Plowsof denied it himself     

> __< s​pirobel:kernal.eu >__ anyway     

> __< s​pirobel:kernal.eu >__ who cares     

> __< d​iego:cypherstack.com >__ In other words, the changing of the terms as it were is a slight, but understandable, misread of the situation     

> __< s​yntheticbird:monero.social >__ you can be helpful and unprofessional at once. I've good impressions of ofrnxmr since he is unbanned, but I do understand fear from some, only time will make ofrn either right or wrong imo     

> __< s​pirobel:kernal.eu >__ lets move forward     

> __< o​frnxmr:monero.social >__ Xmrscott >> show evidence of me threatening people's lives, or buzz off     

> __< x​mrscott:monero.social >__ I'm good, I know what I'm talking about as does Diego     

> __< s​yntheticbird:monero.social >__ wat     

> __< s​pirobel:kernal.eu >__ lets move on to something productive     

> __< s​pirobel:kernal.eu >__ next topic     

> __< o​frnxmr:monero.social >__ We were cool _before_ i was banned too. Idk where your bad impressions came from     

> __< s​yntheticbird:monero.social >__ > 1: guys he is bad     

> __< s​yntheticbird:monero.social >__ > 2: no its wrong I didn't planned a whole revolution and scheme to undermine the credibility of a project through the use of X.     

> __< s​yntheticbird:monero.social >__ Some context would be appreciated     

> __< d​iego:cypherstack.com >__ Regardless, the scope of this proposal is pretty tight. Work on site. Work on docs. etc. And there's no m0 so it's milestone based.     

> __< o​frnxmr:monero.social >__ You cant publicly claim i threaten peoples lives and then casually forget to prove it     

> __< plowsof >__ Rucknium has an upvote on the proposal, which still stands after the merge of bsx      

> __< d​iego:cypherstack.com >__ I don't see an issue giving it a shot and if it doesn't pan out no money is lost.     

> __< 0​xfffc:monero.social >__ ^ this.     

> __< s​yntheticbird:monero.social >__ cake wallet telemetry episode mostly     

> __< d​iego:cypherstack.com >__ if this would have had an m0 I would have given a no here, but it doesn't     

> __< o​frnxmr:monero.social >__ What about it? They fixed it _and_ i worked on fixing their nodes     

> __< x​mrscott:monero.social >__ It's rubberstamping paying out someone Core knows is engaging in malicious actions     

> __< o​frnxmr:monero.social >__ Show me the malicious actions     

> __< d​iego:cypherstack.com >__ Combining the comments on the proposal and this meeting, there seems to be a slight lean toward the positive here.     

> __< s​yntheticbird:monero.social >__ the language being used and insisting on allegations was *unprofessional*. I don't say you weren't right at first, I just say you acted *unprofessional*. But from my pov seeing you being helpful around show me you can be *professional*.     

> __< o​frnxmr:monero.social >__ Youre the one who mutes and bans contributors AND lies about them being malicious, including lies about death threats. You have a _personal_ issue. Leave your fantasia out of this     

> __< d​iego:cypherstack.com >__ Alright, let's stay on topic here     

> __< s​yntheticbird:monero.social >__ alr sry     

> __< d​iego:cypherstack.com >__ we're discussing the proposal. Scott has given his no, and a reason why. Several others have given their yes with some reasoning.     

> __< p​lowsof:matrix.org >__ we have 2 proposals to touch on before the hour     

> __< s​yntheticbird:monero.social >__ I'm just wondering if it is in the standard for people to have two CCS at the same time ?     

> __< d​iego:cypherstack.com >__ With (my personal) read of the consensus being a slight community lean toward yes     

> __< o​frnxmr:monero.social >__ https://matrix.to/#/!oyXaDZnhLzQECPWmIt:monero.social/$W6KdLWXVwHZPDWiXjbMdqRd62vKDC-1wdRbmP_zw9IU?via=monero.social&via=matrix.org&via=frei.chat     

> __< p​lowsof:matrix.org >__ it has happened / is happening (multiple) so its nothing new     

> __< d​iego:cypherstack.com >__ This is not unheard of.     

> __< r​ottenwheel:kernal.eu >__ I am against. For the record.     

> __< d​iego:cypherstack.com >__ Next proposal     

> __< p​lowsof:matrix.org >__ thanks for feedback all, onto:     

> __< o​frnxmr:monero.social >__ (I still want an apology)     

> __< p​lowsof:matrix.org >__ d. [hinto-janai full-time work on Cuprate (3 months)](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/484)     

> __< d​iego:cypherstack.com >__ I don't have much to add here. The cuprate people popped in during my absence.     

> __< s​yntheticbird:monero.social >__ merge obviously     

> __< p​lowsof:matrix.org >__ hinto doesnt appear to be here, and mentioned needing.taking a break after completing the previous one     

> __< r​ottenwheel:kernal.eu >__ In favor. ++     

> __< s​yntheticbird:monero.social >__ hinto is so perfect they could be a spy     

> __< p​lowsof:matrix.org >__ i presume he has the blessings of the cuprate workgroup , just need to leave some updoots     

> __< s​pirobel:kernal.eu >__ xmrscott did more malicious actions than ofrnxmr     

> __< s​pirobel:kernal.eu >__ thats all i am gonna say on this matter     

> __< s​pirobel:kernal.eu >__ next     

> __< d​iego:cypherstack.com >__ on topic guys     

> __< p​lowsof:matrix.org >__ and 0xfffc 's proposal as we're about to his the house (who is/was in attendance of this meeting)     

> __< p​lowsof:matrix.org >__ e. [0xfffc 3 months full-time dev.](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/487)     

> __< p​lowsof:matrix.org >__ s/his the house/hit the hour     

> __< s​yntheticbird:monero.social >__ im against     

> __< s​yntheticbird:monero.social >__ refusing the merge     

> __< s​yntheticbird:monero.social >__ so im for merging :)     

> __< p​lowsof:matrix.org >__ yeah can you not gordan ramsey with peoples livelihoods     

> __< o​frnxmr:monero.social >__ +1 merge     

> __< d​iego:cypherstack.com >__ seems chilil     

> __< o​frnxmr:monero.social >__ Honorable mention: spirobels kuno proposal for wallet work     

> __< p​lowsof:matrix.org >__ unless there is more feedback, i wanted to share the FCMP++ wip PR by jberman https://github.com/monero-project/monero/pull/9436 and tobtohts bootstrappable version wow https://github.com/monero-project/monero/pull/9440     

> __< o​frnxmr:monero.social >__ https://kuno.anne.media/fundraiser/07nm/     

> __< p​lowsof:matrix.org >__ the link with more info here: https://kuno.anne.media/fundraiser/07nm/     

> __< l​ederstrumpf:matrix.org >__ nice sneaking in more rust with ffi :)     

> __< o​frnxmr:monero.social >__ Related to tobtohts bootstrappable fcmp, current proposal fkr bootstrappable      

> __< o​frnxmr:monero.social >__ https://github.com/monero-project/monero/pull/8929     

> __< s​yntheticbird:monero.social >__ Rust take over the world     

> __< 0​xfffc:monero.social >__ This one I am actively reviewing. Will be finished soon.     

> __< o​frnxmr:monero.social >__ Spirobel, can you give a quick explanation of your proposal?     

> __< s​pirobel:kernal.eu >__ this might become a CCS. no idea why people dont donate to the kuno. Clear it is not a scam. lots of references + prior work / donations for community.     

> __< s​yntheticbird:monero.social >__ i forgot kuno existed. clearly not heard a lot of people experiencing it. unknown to most, unheard by most     

> __< s​pirobel:kernal.eu >__ TLDR: wallet rpc is an unmaintained turd. While building the browser walllet i found the monero-js (now monero-ts) to have limitations.     

> __< s​pirobel:kernal.eu >__ here are my references:      

> __< s​pirobel:kernal.eu >__ References:     

> __< s​pirobel:kernal.eu >__ https://x.com/spirobel/status/1672479215512588288 - found and reported a "pay what you want" vulnerability in AcceptXMR     

> __< s​pirobel:kernal.eu >__ https://x.com/spirobel/status/1595949928634667008 - open sourced a Patreon like tool for Monero     

> __< s​pirobel:kernal.eu >__ https://x.com/spirobel/status/1596299822516285440 - open sourced a merchant focused wallet-rpc tool     

> __< s​pirobel:kernal.eu >__ https://www.youtube.com/watch?app=desktop&v=4DLcsQ45zoE - implemented a Monero Browser wallet extension     

> __< s​pirobel:kernal.eu >__ Contact: twitter.com/spirobel     

> __< s​pirobel:kernal.eu >__ i know this codebase and have ideas on how to make it better so it also works with remote daemons / spotty connection     

> __< r​ottenwheel:kernal.eu >__ "Implemented a Monero Browser wallet extension" is just a video of something we never saw even a repository of.     

> __< s​pirobel:kernal.eu >__ but this time i want to see support first. spent enough of my own time and money (500 dollars donation to anarkio for example) on monero     

> __< s​pirobel:kernal.eu >__ no free work anymore     

> __< r​ottenwheel:kernal.eu >__ Neither of the prior two items were audited, nor deployed in any environment yet.     

> __< r​ottenwheel:kernal.eu >__ Barely any "references" besides a piece of paper on a pole in Afghanistan. 🤣     

> __< s​pirobel:kernal.eu >__ critque someone else I am not going to engage with you. non of your critques are in good faith     

> __< p​lowsof:matrix.org >__ thanks for sharing spirobel.  i think we can put an end to the meeting and continue as usual. thanks all for attending     


Automated by [this](https://github.com/plowsof/post-libera-meeting-logs)
